File: Atcoder_#363/C/a_piling_up.c
#include <stdio.h>

int main(void) {
    int r;
    int next_level;
    int required_increase;
    
    scanf("%d", &r);

    if (r <= 99) {
        next_level = 100;
    } else if (r <= 199) {
        next_level = 200;
    } else {
        next_level = 300;
    }

    required_increase = next_level - r;
    
    printf("%d\n", required_increase);
    
    return 0;
}

File: Atcoder_#363/javascript/a_piling_up.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('', (r) => {
  const R = parseInt(r, 10);
  let next_level;

  if (R <= 99) {
    next_level = 100;
  } else if (R <= 199) {
    next_level = 200;
  } else {
    next_level = 300;
  }

  const required_increase = next_level - R;
  console.log(required_increase);

  rl.close();
});
 